docs/reference/migration/migrate_5_0.asciidoc

```diff
@@ -24,7 +24,7 @@ version of Elasticsearch before 2.0.
 =========================================
 
 Indices created in Elasticsearch 1.x or before will need to be reindexed with
-Elasticsearch 2.x in order to be readable by Elasticsearch 5.x. It is not
+Elasticsearch 2.x or 5.x in order to be readable by Elasticsearch 5.x. It is not
```

Comment:
Ok, that makes more sense now. :) I personally don't think it's necessary -- I would hope a reader would click on the link two sentences later pointing them to more in-depth info on how they will need to re-index. This PR is more for the sake of accuracy.